src/mailing/sender.js

```diff
@@ -1,6 +1,8 @@
 const nodemailer = require('nodemailer');
 const Email = require('email-templates');
 const path = require('path');
+const ejs = require('ejs');
+require('dotenv').config();
```

Comment:
I don't think this is needed. When this file is run by the server, the dotenv library will already have been imported in `expressServer.js`

Reply:
Removed import ejs , commit d5d367fe09e202e201a1512da868012240df2b8b

Reply:
we can also remove `require('dotenv').config();`